scootapi/testhelpers/generators.go

```diff
@@ -32,7 +32,7 @@ func GenJobDefinition(rng *rand.Rand) *scoot.JobDefinition {
 func GenTask(rng *rand.Rand) *scoot.TaskDefinition {
 
 	cmd := scoot.NewCommand()
-	cmd.Argv = []string{"sleep", "1"}
+	cmd.Argv = []string{"sleep 500", "complete 0"}
```

Comment:
Right, but this PR could do more to be end-to-end. Unless I'm misreading, we use os/execer and by redefining tasks here, you'll break anything that currently runs scootapi.smoketest.